app/controllers/custom_attributes_controller.rb

```diff
@@ -35,35 +35,36 @@ def choice
   end
 
   private
-  def check_type_param
-    redirect_to root_path if params[:type].blank?
-  end
 
-  def update_existing_attributes(params)
-    attributes = CustomAttribute.attributes_for(current_user.company, params[:type])
+    def check_type_param
```

Comment:
I use sublime text 3. I used to have a configuration in sublime to auto-indent, But it wasn't consistent with different cases. Currently I do it manually. Rails and Padrino use this style to give private/protected methods more visibility..